packages/aws-lambda-otel-extension/opt/otel-extension/external/lambda-apis/runtime-api.js

```diff
@@ -24,6 +24,22 @@ async function getEventData() {
   };
 }
 
+async function getRuntimeEventData() {
```

Comment:
I see this method is added, while we already have `getEventData`. Why we do not reuse `getEventData` (which if I see correctly is not used anywhere, so if not used should be removed)